frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -112,6 +114,21 @@ export default function CourseDescription({ structure }) {
     }
   }, [id]);
 
+  const [collapseRequirements, setCollapseRequirements] = useState(false);
+  const [collapseOverview, setCollapseOverview] = useState(false);
+  const [collapseDone, setCollapseDone] = useState(false);
+  const [collapseDirectUnlock, setCollapseDirectUnlock] = useState(false);
+  const [collapseIndirectUnlock, setCollapseIndirectUnlock] = useState(true);
+
+  const toggleCollapse = (state, setState) => {
```

Comment:
Is this `toggleCollapse` being used anywhere in this file?

Reply:
Was being used originally but the responsibility was moved into the component when cleaning up code.
Resolved.